### PR TITLE
Feature/bi 3690

### PR DESCRIPTION
**Pull request by sapphireduffy**

### CODE REVIEW

File: src/paths.ts [hunk 1]
```diff
@@ -4,3 +4,4 @@ export const HEALTHCHECK_URI = `${ROOT_URI}/healthcheck`
 
 // Pages
 export const FORM_PAGE_URI = `${ROOT_URI}/form`
+export const WHOTOTELL_URI = `${ROOT_URI}/who-to-tell`
```
Comment (mateuszc-kainos) [inline src/paths.ts:7]: Perhaps WHO_TO_TELL_URI would be a better name?

File: src/views/landing.njk [hunk 2]
```diff
@@ -7,36 +7,36 @@
...
+  <main class="govuk-main-wrapper">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+       <h1 class="govuk-heading-xl">{{ serviceName }}</h1>
+       <p class="govuk-body-l">Use this service to apple to close a company, also known as striking a company off the register or dissolution.</p>
+       <p class="govuk-body">It costs £8 to close a company and you'll need:</p>
+        <ul class="govuk-list govuk-list--bullet">
+          <li>the company number</li>
+          <li>a Companies House password - you'll be able to register if you don't have one</li>
+         <li>the company authentication code</li>
```
Comment (mateuszc-kainos) [inline src/views/landing.njk:20]: indentation

File: src/views/landing.njk [hunk 3]
```diff
@@ -7,36 +7,36 @@
...
+         <li>an email address or each director if the company has more than one</li>
```
Comment (mateuszc-kainos) [inline src/views/landing.njk:21]: 'an email address for each director if the company has more than one'
also, indentation

**Comment by mateuszc-kainos** [comment]
Good job!

File: src/paths.ts (hunk 1 above)
Comment (sapphireduffy) [reply src/paths.ts:7]: Ok, This is what Carl named it so I just copied what he had - I'll let him know to change it too! :)

File: src/views/landing.njk [hunk 4]
```diff
@@ -7,36 +7,36 @@
...
+          <li>a Companies House password - you'll be able to register if you don't have one</li>
+          <li>the company authentication code</li>
+          <li>an email address for each director if the company has more than one</li>
+          <li>a credit or debit card</li>
+        </ul>
+        <form method="post">
+         {{
+            govukButton({
+              text: 'Start now',
+              href: "/who-to-tell`",
```
Comment (peadarkelly) [inline src/views/landing.njk:28]: there's a rogue ' here

File: src/views/landing.njk [hunk 5]
```diff
@@ -7,36 +7,36 @@
...
-      ]
-    })
-  }}
-{% endblock %}
-
 {% block content %}
-  <div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
-      <h1 class="govuk-heading-xl">{{ serviceName }}</h1>
-      <p class="govuk-body-l">Use this service to get your NodeJS application up and running</p>
```
Comment (peadarkelly) [inline src/views/landing.njk:26]: Unit tests in `landing.controller.test.ts` will need to be updated to match the new content. Also think we need to add coverage for clicking the start button / `post` method.

File: src/views/landing.njk (hunk 4 above)
Comment (sapphireduffy) [reply src/views/landing.njk:28]: Good spot!

File: src/views/landing.njk [hunk 6]
```diff
@@ -7,36 +7,36 @@
...
+          <li>a Companies House password - you'll be able to register if you don't have one</li>
+          <li>the company authentication code</li>
+          <li>an email address for each director if the company has more than one</li>
+          <li>a credit or debit card</li>
+        </ul>
+        <form method="post">
+         {{
+            govukButton({
+              text: 'Start now',
+              href: "/who-to-tell",
```
Comment (peadarkelly) [inline src/views/landing.njk:28]: wait, how does this work? Does it call the POST method in the landing controller, or navigate directly to the Who to tell page? If it calls the controller, then remove the href here.

I'm surprised the href works too. Would've thought it needed to be `/close-a-company/who-to-tell`

File: test/controllers/landing.controller.test.ts [hunk 7]
```diff
@@ -2,19 +2,26 @@ import 'reflect-metadata'
...
 
-    assert.include(res.text, 'Use this service to get your NodeJS application up and running')
+  describe('POST request', () => {
+    it('should redirect to the who to tell screen upson submission', async () => {
+    await request(app).post(ROOT_URI).expect(MOVED_TEMPORARILY).expect('Location', WHO_TO_TELL_URI)
+    })
   })
-})
+
+
```
Comment (peadarkelly) [inline test/controllers/landing.controller.test.ts:27]: watch extra lines at the end

File: test/controllers/landing.controller.test.ts [hunk 8]
```diff
@@ -2,19 +2,26 @@ import 'reflect-metadata'
...
+    assert.include(res.text, 'Use this service to apply to close a company, also known as striking a company off the register or dissolution.')
+    })
+  })
 
-    assert.include(res.text, 'Use this service to get your NodeJS application up and running')
+  describe('POST request', () => {
+    it('should redirect to the who to tell screen upson submission', async () => {
+    await request(app).post(ROOT_URI).expect(MOVED_TEMPORARILY).expect('Location', WHO_TO_TELL_URI)
+    })
   })
```
Comment (peadarkelly) [inline test/controllers/landing.controller.test.ts:25]: I think this should be:
```
describe('LandingController', () => {
  describe('GET request', () => {
    it('should...', async () => {
      ...
    })
  })

  describe('POST request', () => {
    it('should...', async () => {
      ...
    })
  })
})
```

**Pull request merged**